prometheus-to-sd/translator/prometheus.go

```diff
@@ -66,13 +77,8 @@ func getPrometheusMetrics(config *config.SourceConfig) (*PrometheusResponse, err
 }
 
 func doPrometheusRequest(url string, auth config.AuthConfig) (resp *http.Response, err error) {
-	// Allow insecure https connections
 	client := &http.Client{
```

Comment:
This still creates a new HTTP client for each request. Can we have the client initialized just once?

Reply:
Done.